automatic/nuget.commandline/update.ps1

```diff
@@ -21,13 +21,33 @@ function global:au_GetLatest {
 
   $versions = $json."nuget.exe" | Sort-Object uploaded -Descending
   $preRelease = $versions | ? stage -EQ 'EarlyAccessPreview' | select -First 1
+  $stableRelease46 = $versions | ? stage -EQ 'ReleasedAndBlessed' | ? version -like '4.6.*' | select -First 1
+  $stableRelease47 = $versions | ? stage -EQ 'ReleasedAndBlessed' | ? version -like '4.7.*' | select -First 1
+  $stableRelease48 = $versions | ? stage -EQ 'ReleasedAndBlessed' | ? version -like '4.8.*' | select -First 1
+  $stableRelease49 = $versions | ? stage -EQ 'ReleasedAndBlessed' | ? version -like '4.9.*' | select -First 1
```

Comment:
Any reason why you just didn't iterate through the different releases?
I'm thinking that would be better in the long run instead of manually adding a new stream for every new release.